packages/popover/src/vaadin-popover.js

```diff
@@ -445,14 +445,14 @@ class Popover extends PopoverPositionMixin(
   connectedCallback() {
     super.connectedCallback();
 
-    document.addEventListener('click', this.__onGlobalClick, true);
+    document.documentElement.addEventListener('click', this.__onGlobalClick, true);
```

Comment:
**question:** I wonder if there is a particular reason for using `document.documentElement` instead of `document` in the overlay?

Reply:
This was needed to workaround a Firefox issue: https://github.com/vaadin/vaadin-overlay/pull/148
The corresponding [bug is still open](https://bugzilla.mozilla.org/show_bug.cgi?id=990614#c4) so maybe we should keep it this way for now.